luxonis_train/attached_modules/visualizers/utils.py

```diff
@@ -145,6 +148,158 @@ def draw_bounding_box_labels(img: Tensor, label: Tensor, **kwargs) -> Tensor:
     return draw_bounding_boxes(img, bboxs, **kwargs)
 
 
+def draw_obounding_box(
+    img: Tensor, obbox: Tensor | np.ndarray, **kwargs
+) -> Tensor:
+    """Draws oriented bounding box (obb) labels on an image.
+
+    @type img: Tensor
+    @param img: Image to draw on.
+    @type obbox: Tensor
+    @param obbox: Oriented bounding box. The shape should be
+        (n_instances, 8) or (n_instances, 5), where the last dimension
+        is (x1, y1, x2, y2, x3, y3, x4, y4) or (xc, yc, w, h, r).
+    @type kwargs: dict
+    @param kwargs: Additional arguments to pass to
+        L{draw_obounding_boxes}.
+    @rtype: Tensor
+    @return: Image with bounding box labels drawn on.
+    """
+    _, H, W = img.shape
+    # The conversion below is needed for fitting a rectangle to the 4 label points, which can form
+    # a polygon sometimes
+    if obbox.shape[-1] > 5:
+        obbox = xyxyxyxy2xywhr(obbox)  # xywhr
+    bboxs_2 = xywhr2xyxyxyxy(obbox)  # shape: (bs, 4, 2)
+    if isinstance(bboxs_2, np.ndarray):
+        bboxs_2 = torch.tensor(bboxs_2)
+    if bboxs_2.numel() == 0:
+        raise ValueError
+    bboxs = bboxs_2.view(bboxs_2.size(0), -1)  # x1y1x2y2x3y3x4y4
+    bboxs[:, 0::2] *= W
+    bboxs[:, 1::2] *= H
+    return draw_obounding_boxes(img, bboxs, **kwargs)
+
+
+def draw_obounding_boxes(
```

Comment:
I suppose this is mostly copied from `torchvision`, it should be updated to be consistent with the rest of the codebase (remove python 3.8 typing, `Tensor` instead of `torch.Tensor` etc)

Reply:
Additionally, since we're reimplementing the `torchvision` functions anyways, we can just make a single function that will be able to handle both normal and oriented bboxes instead of having two almost identical functions.